yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryCommand.java

```diff
@@ -114,6 +113,9 @@ private QueryCommand(int lineNumber, @Nonnull QueryInterpreter interpreter, @Non
         }
         this.queryInterpreter = interpreter;
         this.queryConfigs = configs;
+        Assert.thatUnchecked(queryConfigs.stream()
+                .filter(config -> config instanceof QueryConfig.SkipConfig).count() < 2,
+                "Too many skips in a Query " + lineNumber);
```

Comment:
I might not understand it fully, but it looks like multiple things are happening at different places that are aimed towards validating if the `QueryCommand` should run or not. 

There is a `yamsql` "compile-time" check of the supported version (and logging) happening in `parse(...)`, and there is also a runtime check and logging happening during `execute(...)`. We can maybe avoid duplication by having a `NoOp` command, which is similar in essence to the `NoOpBlock` and `NoOpConfig`.

Reply:
There might also be some value in ensuring that the `supportedVersion` is the first config in the series of configs. Else, it gives a notion of configs that defined before the `supportedVersion` would run for all the versions while the ones defined after are version protected.